.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/KDAB/android_openssl
 [submodule "libs/qmlglsink/gst-plugins-good"]
 	path = libs/qmlglsink/gst-plugins-good
-	url = https://github.com/mavlink/gst-plugins-good.git
```

Comment:
Do we need this, or can we update the mavlink one with the changes required?

Reply:
It would be great to merge my changes to `mavlink/gst-plugins-good`
I changed to mine, to have a quick path fix my fixes. But, bear in mind that not so long ago it was patricelectric's repo here :) 
So, shall I create a PR to  `mavlink/gst-plugins-good`? Will you or sbd else approve it quite fast?
Shall we wait with current PR until new PR into  `mavlink/gst-plugins-good` goes through?

Maybe we could push with my submodule, until I update  `mavlink/gst-plugins-good`?
This would be something that opens possibility for others to test video stream for bugs. Also non-linux platforms are still do be fixed. I can take Windows, and then eventually Android. I have no Apple hardware (both macbook nor iphone)

Reply:
I should be able to approve and merge it, yes.

Reply:
@julianoes please create a new branch in `mavlink/gst-plugins-good`, it might be for example `qgroundcontrol_qt6` or anythyng else, like `qt6_qmlglsink` 
I have to create PR to existing branch. 
`qgroundcontrol` cannot be upgraded to qt6, as it's still  used by qt5, QGC 4.x version. We cannot break it.

Reply:
Although, as long as the submodule points to the previous commit it should be fine.

Reply:
Ah, unless you want to branch from upstream.

Anyway, I created a branch for you:
https://github.com/mavlink/gst-plugins-good/tree/qgroundcontrol-qt6

And I'll give you permission to the repo, so you can directly push to that one.

Reply:
> `` cannot be upgraded to qt6, as it's still used by qt5, QGC 4.x version. We cannot break it.

You shouldn't branch gst-plugins-good. The submodules in the Stable branches of QGC will stay pointed to the place before any mods to plugins-good were made for Qt6. Hence they will build file even though plugins-good has moved on.

You should instead tag plugins-good with a tag like "pre-qt6". Then in the unlikely event that a Stable build needs a fix in plugins-good, that is when plugins-good can be branched from that specific  tag. And then the Stable builds would reference that branch which has the fixes.

Reply:
@DonLakeFlyer  you're right, this is fine, but I'm not comfortable with dangling references. I'd like to have branches, as we have branches with current and stable release.